test/testpy/__init__.py

```diff
@@ -71,13 +71,17 @@ def GetCommand(self):
       # inspector related tests). Also, if there is no ssl support the options
       # '--use-bundled-ca' and '--use-openssl-ca' will also cause a similar
       # failure so such tests are also skipped.
-      if ('--inspect' in flag[0] or \
-          '--use-bundled-ca' in flag[0] or \
-          '--use-openssl-ca' in flag[0]) and \
-          self.context.v8_enable_inspector == 0:
-        print('Skipping as node was configured --without-ssl')
+      if (any(flag.startswith('--inspect') for flag in flags) and
+          not self.context.v8_enable_inspector):
+        print('Skipping as node was compiled without inspector support')
+      elif (('--use-bundled-ca' in flags or
+          '--use-openssl-ca' in flags or
+          '--tls-v1.0' in flags or
+          '--tls-v1.1' in flags) and
```

Comment:
Nit: Perhaps all these flags can be maintained in a List and using `any` to figure out if they are set would be better, I believe.

Reply:
You mean checking `if set(flags) & set(blacklist):`? I considered suggesting that but either way is fine to me, really. It's not like there are a ton of flags to blacklist.

Reply:
Yup, something like that. We can just reduce a few lines. I am also okay with either. This was a nit anyway ;-)

Reply:
I was not sure if it was worth the effort here as there are not that many flags at the moment, but I'd be happy to add this if any more flags are added. Thanks